hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieLogFormatVersion.java

```diff
@@ -80,18 +80,15 @@ public boolean hasHeader() {
 
   @Override
   public boolean hasFooter() {
-    switch (super.getVersion()) {
-      case DEFAULT_VERSION:
-        return false;
-      case 1:
-        return true;
-      default:
-        return false;
-    }
+    return hasLogFooterAndBlockLength();
```

Comment:
Can you please limit a PR to a particular issue ? For example, you can just use this PR to change Schema.parse to new Schema.Parser(). The other changes could be part of another PR (which should have it's own meaning) but at the same time necessitating a change.

Reply:
@ambition119